README.md

```diff
@@ -15,6 +16,12 @@ The kartograafr application...
 * can be easily updated by changing its configuration course page in Canvas,
 * allows support staff to update its configuration via Canvas without requiring server sysadmin access
 
+## Conventions:
+* The git workflow is to use feature branches for development.
```

Comment:
I know you said that this and several other TL projects use the "feature branch" git workflow rather than the "gitflow" one.  Maybe it's because I've been working mostly on "gitflow"-based projects, but I would appreciate a summary (or a link to one) of the "feature branch" workflow here.